### PR TITLE
docs(gen): Update contributing guidelines with fork instructions

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -7,19 +7,23 @@ title: Contributing
 
 ## Who can contribute?
 
-Anyone who has a GitHub account can contribute to the documentation content.
+Anyone who has a GitHub account can contribute to the documentation content:
+
+- Users within the Scaleway organization can open pull requests directly in the docs-content repository.
+
+- External users (i.e. not part of the Scaleway organization) must fork the repository and open a pull request from the forked repository.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:14]:
```suggestion
- External users (i.e. not part of the Scaleway organization) **must fork the repository and open a pull request from the forked repository**.
```

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -54,6 +58,54 @@ Anyone who has a GitHub account can contribute to the documentation content.
...
+
+1. Access the [docs-content repository](https://github.com/scaleway/docs-content/), and click **Fork** in the top right corner. A form appears.
+
+2. Keep the default values in the form, and click **Create Fork**.
+
+3. Make sure you have identified the [type of content](https://github.com/scaleway/docs-content#what-is-the-scaleway-documentation-platform) that you want to write (e.g. tutorial, API/CLI documentation, additional content or troubleshooting).
+
+4. Create your branch locally. All branch names must comply with the following naming convention: `[source]-[action]-[product]`.
+
+    - `source`: Either `int` (for internal Scaleway contributors) or `ext` (for external contributors). If you are not a Scaleway staff member, please use `ext`.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:75]:
```suggestion
    - `source`: Either `int` (for internal Scaleway contributors) or `ext` (for external contributors). If you are not a Scaleway staff member, use `ext`.
```

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -54,6 +58,54 @@ Anyone who has a GitHub account can contribute to the documentation content.
...
+
+    - `action`: The action you will perform in the documentation you wish to work on, described in 3 letters or less. Example: `add` (for adding content), `rm` (for removing content) or `fix` (for when you wish to fix typos).
+
+    - `product`: the name of the product that corresponds to the documentation page you will edit.
+
+    :information_source: **Example:**  
+      in the branch `ext-add-instances`, an external contributor will add information to a documentation page of the Instances product category.
+
+4. Make your edits to the documentation. Refer to the [writing guidelines](#writing-guidelines) below to ensure your changes comply with our style guide.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:84]:
```suggestion
4. Make your edits to the documentation. Refer to the [writing guidelines](#writing-guidelines) below to ensure your changes comply with our guidelines. 
```

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -54,6 +58,54 @@ Anyone who has a GitHub account can contribute to the documentation content.
...
+
+5. Run `git commit -m "type(module): description with some words <BRANCH_NAME>"`. Refer to the [commit naming conventions](#commit-conventions) below to properly name your commit.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:86]:
```suggestion
5. Run `git commit -m "type(module): description of what you performed <BRANCH_NAME>"`. Refer to the [commit naming conventions](#commit-conventions) below to properly name your commit.
```

File: docs/CONTRIBUTING.md [hunk 5]
```diff
@@ -54,6 +58,54 @@ Anyone who has a GitHub account can contribute to the documentation content.
...
+
+  - base: `main`
+
+  - head repository: `<your-account>/docs-content`
+
+  - compare: `<your-branch-with-changes>`
+
+8. Click **Create pull request**.
+
+    All PR names must comply with the following naming convention: `docs($PRODUCTNAME): $FIXEXPLANATION BRANCHNAME`.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:102]:
```suggestion
    All Pull Request names must comply with the following naming convention: `docs(<PRODUCTNAME>): <ACTION> <EXPLANATION> <BRANCHNAME>`.
```

File: docs/CONTRIBUTING.md [hunk 6]
```diff
@@ -54,6 +58,54 @@ Anyone who has a GitHub account can contribute to the documentation content.
...
+
+:warning: **Important**  
+  All PRs are reviewed and approved by the Scaleway Product Documentation team before being merged by them.
```
Comment (nerda-codes) [inline docs/CONTRIBUTING.md:105]:
```suggestion
  All Pull Requests are reviewed and approved by the Scaleway Product Documentation team before being merged by them.
```

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (nerda-codes) [reply docs/CONTRIBUTING.md:14]: Can we link the specific procedure for internal users in the Scaleway orga and the one for users not in the orga here? So people don't have to scroll through the first procedure if they can't use it

**Review by ldecarvalho-doc — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**